### PR TITLE
fix overwrite runtime query.

### PR DESCRIPTION
**Pull request by MohammedSDev**
# Pull Request Template

Thank you for contributing! Please take a moment to review our [**contributing guidelines**](https://github.com/AniTrend/retrofit-graphql/blob/master/CONTRIBUTING.md)
to make the process easy and effective for everyone involved.

**Please open an issue** before embarking on any significant pull request, especially those that
add a new library or change existing tests, otherwise you risk spending a lot of time working
on something that might not end up being merged into the project.

Before opening a pull request, please ensure:

- [ ] You have followed our [**contributing guidelines**](https://github.com/AniTrend/retrofit-graphql/blob/master/CONTRIBUTING.md)
- [ ] double-check your branch is based on `develop` and targets `develop`
- [ ] Pull request has tests
- [ ] Code is well-commented, linted and follows project conventions
- [ ] Documentation is updated (if necessary)
- [ ] Description explains the issue/use-case resolved
- [ ] I did not commit files that are excluded in the .gitignore file (Happens if you stage files with Android Studio)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

**IMPORTANT**: By submitting a patch, you agree to allow the project
owners to license your work under the terms of the [Apache License](https://github.com/AniTrend/retrofit-graphql/blob/master/LICENSE.md).

### CODE REVIEW

**Comment by wax911** [comment]
Thank you for your pull request @MohammedSDev I see you did not do the following:
image

TLDR; 
1. You should always create an issue first before creating a pull request to discuss the problem, otherwise you spend time working something that may already be in the works or allocated to someone else
2. Your pull request should **ALWAYS** target the _develop_ branch
